src/components/Home.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import BookList from './BookList';
+import BookForm from './BookForm';
+
+const Home = () => {
+  const [books, setBooks] = useState([]);
+
+  const handleCreateBook = (title) => {
+    const newBook = { id: Date.now(), title };
+    setBooks((prevBooks) => [...prevBooks, newBook]);
+  };
```

Comment:
- [ ] Currently, a newly added book only includes the attributes "title" and "id." However, adhering to the project requirements, it's essential to expand the new book's attributes to encompass "id," "title," and "author name." Please ensure alignment with the stipulated project specifications.
- [ ] Moreover, for adding a new book to the Redux store, I recommend utilizing the book reducer and the dispatch method. This approach facilitates the integration of the new book into the Redux store while maintaining consistency with the project's guidelines. Your attention to detail and adherence to the project's outlined directives are greatly appreciated.

Reply:
- Refactor all files to use Redux in React components to solve bugs.